composer.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "utopia-php/detector",
+  "description": "A simple library to detect popular runtimes, frameworks and package managers etc.",
```

Comment:
I would avoid mentions of specific adapters, AI can help to prepare a copy, you can validate with me over Discord